### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by cuishuang**

### CODE REVIEW

File: crates/ide-db/src/generated/lints.rs [hunk 1]
```diff
@@ -10462,7 +10462,7 @@ if the `never_type` is enabled."##,
     },
     Lint {
         label: "clippy::empty_line_after_doc_comments",
-        description: r##"Checks for empty lines after documenation comments."##,
+        description: r##"Checks for empty lines after documentation comments."##,
```
Comment (lnicola) [inline crates/ide-db/src/generated/lints.rs:10465]: These are or were misspelled in clippy, let's drop them from here.

Comment (cuishuang) [reply crates/ide-db/src/generated/lints.rs]: Thanks. Done

Comment (lnicola) [reply crates/ide-db/src/generated/lints.rs]: There's another one in the same file 😀

Comment (cuishuang) [reply crates/ide-db/src/generated/lints.rs]: Sorry for my lack of rigor. 

Already edited

**Pull request merged**